src/main/resources/Help/WebHome.xml

```diff
@@ -104,60 +104,60 @@
+  'title': $services.localization.render('help.templates.title'),
+  'description':  $services.localization.render('help.templates.description'),
   'documentation': 'Help.Templates.WebHome',
   'examples': $xwiki.getURL('Help.Templates.WebHome', 'view', $NULL, 'HExamples')
 }, {
   'icon': 'fa fa-file-excel-o',
-  'title': 'Macros',
-  'description': 'Macros are reusable functionality that can be integrated inside a page.',
+  'title': $services.localization.render('help.macros.title'),
+  'description':  $services.localization.render('help.macros.description'),
   'documentation': 'Help.Macros.WebHome',
   'examples': $xwiki.getURL('Help.Macros.WebHome', 'view', $NULL, 'HExamples')
 }, {
   'icon': 'fa fa-files-o',
-  'title': 'Applications',
-  'description': 'Applications allow to group related structured pages together.',
+  'title': $services.localization.render('help.applications.title'),
+  'description':  $services.localization.render('help.applications.description'),
   'documentation': 'Help.Applications.WebHome',
   'examples': $xwiki.getURL('Help.Applications.WebHome', 'view', $NULL, 'HExamples')
 }, {
   'icon': 'fa fa-sitemap',
-  'title': 'Content Organization',
-  'description': 'Organize your content in hierarchical pages, grouped in several (sub)wikis.',
+  'title': $services.localization.render('help.organization.title'),
+  'description':  $services.localization.render('help.organization.description'),
   'documentation': 'http://platform.xwiki.org/xwiki/bin/view/Features/ContentOrganization/',
   'examples': $xwiki.getURL('Main.AllDocs', 'view', 'view=tree')
 }, {
   'icon': 'fa fa-history',
-  'title': 'Version Control',
-  'description': 'View previous versions of a page, compare different versions or rollback to any past version.',
+  'title': $services.localization.render('help.history.title'),
+  'description':  $services.localization.render('help.history.description'),
   'documentation': 'http://platform.xwiki.org/xwiki/bin/view/Features/VersionControl',
   'examples': $historyURL
 }, {
   'icon': 'fa fa-download',
-  'title': 'Import / Export',
-  'description': 'Import content from various sources and export it in an open format (XML, HTML, PDF, ODT).',
+  'title': $services.localization.render('help.import.title'),
+  'description':  $services.localization.render('help.import.description'),
   'documentation': 'http://platform.xwiki.org/xwiki/bin/view/Features/Imports'
 }])
 {{/velocity}}
 
 {{velocity}}
-= How to use XWiki? =
+= $services.localization.render('help.title1') =
 
-XWiki offers a wide range of powerful features that will enable you to create, organize and find content. This section will show you how to use the basic features of XWiki.
+$services.localization.render('help.description1')
 
 {{html clean="false"}}
 #display4Cards($howToCards)
 {{/html}}
 
-= Main XWiki features =
+= $services.localization.render('help.title2') =
```

Comment:
So still using translation keys instead of translation pages?

Reply:
In the last discussion, it was mentioned  to use translation pages only for templates. In Template Application, I used translation pages as it was suggested (see: https://github.com/xwiki-contrib/application-templates/pull/4/commits/798aa8f566b75c48b41e02a83cd59af9310ae83b). I would have appreciated to be told from the beginning which is the right way to add translation for this application (there were multiple requests for this application to be translated). I already invested a lot of time working on this, so I would like to know a clear direction in which way this should go.

Reply:
It's indeed the right way to go. We discussed it on the chat and yeah it's better to have translation keys: it avoids a lot of problem when getting translations from non technical users who forgets about some technical characters.